ACE/tests/Native_Condition_Variable_Test.cpp

```diff
@@ -0,0 +1,47 @@
+
+//=============================================================================
+/**
+ *  @file    Native_Condition_Variable_Test.cpp
```

Comment:
Agreed, I will probably change it to something like Condition_Variable_Timeout_Test.